src/pubsub-message.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-env mocha */
+'use strict'
+
+const expect = require('chai').expect
+const isNode = require('detect-node')
+const PubsubMessage = require('../../src/pubsub-message') // eslint-disable-line no-unused-vars
```

Comment:
you shouldn't require them, the should be passed in through the `setup` function